CONTRIBUTING.md

```diff
@@ -62,7 +62,16 @@ included in the project:
    git checkout -b <topic-branch-name>
    ```
 
-4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem to preview locally.
+4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem and dependencies to preview locally:
+
+    ```bash
+    # Install the needed gems
+    gem install jekyll jekyll-sitemap pygments.rb
```

Comment:
That's all nice and dandy if you manage the environment in which you deploy, except you don't - GitHub does.
Not committing the .lock file will ensure if you do build testing using travis that travis will get the latest version as running on GH, rather than the version as *you* use on your local machine.